aiomisc/log/config.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+import sys
+from typing import Union
+
+from .enum import LogFormat
+from .formatter import json_formatter, color_formatter
+from .wrap import wrap_logging_handler
+
+DEFAULT_FORMAT = '%(levelname)s:%(name)s:%(message)s'
+
+
+def create_logging_handler(log_format: LogFormat = LogFormat.color,
+                           date_format=None, **kwargs):
```

Comment:
Missing type annotation:
```python
date_format: Optional[LogFormat]
```

Reply:
No the `Optional` is synonym to `Union[LogFormat, None]`, `None` raises an exception in this case.